docs/details.md

```diff
@@ -0,0 +1,259 @@
+
+**Table of Contents**
+
+<!--ts-->
+   * [elixir-omg applications](#elixir-omg-applications)
```

Comment:
bump comment: look at https://github.com/omisego/elixir-omg/blob/93262f423f08fb180403cdaa9ec14aa4b1cd8432/docs/details.md and see which links are broken. Also some sections are missing.

Alternatives:
 - ditch the toc altogether, but there was an explicit request to have one earlier.
 - use the toc-generating tool from the original `README`

your pick.

EDIT: actually, I strongly think the TOC is superfluous as the doc is much smaller now. However I see the sections structure is a bit garbled. I'll try to suggest changes in the doc.